api/CcsSso.Core.DbMigrations/Scripts/Phase2_Sprint2/202201031045_CountryCode.AddRemove.sql

```diff
@@ -0,0 +1,14 @@
+START TRANSACTION;
+
+UPDATE public."CountryDetails" 
```

Comment:
Are you sure this will replace the current country? (Im not sure about the "WHERE" condition.) and also You need to set the both country name and code for the new records.

Reply:
I had a look into the existing script. Seems like we have the code but not the correct name. Can you confirm this?